typescript/infra/config/environments/mainnet3/warp/configGetters/utils.ts

```diff
@@ -0,0 +1,59 @@
+import assert from 'assert';
+
+import { ChainMap, ChainName, MovableTokenConfig } from '@hyperlane-xyz/sdk';
+import { arrayToObject, objMap } from '@hyperlane-xyz/utils';
+
+import { getRegistry } from '../../../../registry.js';
+import { WarpRouteIds } from '../warpIds.js';
+
+const REBALANCER = '0xa3948a15e1d0778a7d53268b651B2411AF198FE3';
+
+type RebalancingConfig = Required<
+  Pick<MovableTokenConfig, 'allowedRebalancingBridges' | 'allowedRebalancers'>
+>;
+
+export function getRebalancingBridgesConfigFor(
+  deploymentChains: readonly ChainName[],
+  chainsToExclude: readonly ChainName[] = [],
+): ChainMap<RebalancingConfig> {
+  const registry = getRegistry();
+  const mainnetCCTP = registry.getWarpRoute(WarpRouteIds.MainnetCCTP);
+
+  assert(mainnetCCTP, 'MainnetCCTP warp route not found');
+
+  const rebalanceableChains = deploymentChains.filter(
+    (chain) => !chainsToExclude.includes(chain),
+  );
+
+  const cctpBridgesByChain = Object.fromEntries(
+    mainnetCCTP.tokens.map(
+      ({ chainName, addressOrDenom }): [string, string] => {
+        assert(
+          addressOrDenom,
+          `Expected cctp bridge address to be defined on chain ${chainName}`,
+        );
```

Comment:
when would this be undefined?

Reply:
on the type definition it can be null so better be sure it is defined:
![image](https://github.com/user-attachments/assets/798d09bf-31e3-4ea3-ad7c-2f5d8b8bbaad)